src/components/View/index.js

```diff
@@ -257,11 +260,21 @@ export default class View extends Component {
          if (paddingLeft !== 0) currentSizeStyles.paddingLeft = paddingLeft
       }
 
+      if (this.props.maxHeight) {
+         const maxHeight = getAttributeForCurrentSize(this.state.size, this.props.maxHeight)
+         if (maxHeight) currentSizeStyles.maxHeight = maxHeight
+      }
+
       if (this.props.maxWidth) {
          const maxWidth = getAttributeForCurrentSize(this.state.size, this.props.maxWidth)
          if (maxWidth) currentSizeStyles.maxWidth = maxWidth
       }
 
+      if (this.props.minHeight) {
```

Comment:
yep, this is possible and some cases useful. This is essentially like adding `min-height` and `max-height` css values on a div. An example where this might be the case is if you had a UI Card or something similar and it had text of variable length. You might want that card to adjust it's height depending on the length of the text (a la masonry). But you might want to set a minimum height and maximum height for that UI Card and use those params to truncate the text with css.